hack/istio/install-bookinfo-demo.sh

```diff
@@ -161,7 +174,7 @@ if [ "${DELETE_BOOKINFO}" == "true" ]; then
 fi
 
 # If OpenShift, we need to do some additional things
-if [[ "$CLIENT_EXE" = *"oc" ]]; then
+if [[ "${IS_OPENSHIFT}" == "true" ]]; then
```

Comment:
use single-brackets with `==` - no need anymore for the double-brackets (which was needed due to the `= *` feature):
```suggestion
if [ "${IS_OPENSHIFT}" == "true" ]; then
```